### PR TITLE
Move from React to Preact; remove FontAwesome SVGs and rely on icon font, remove IE11 support (and all polyfills)

### PR DESCRIPTION
**Pull request by matmannion**
This is WIP at the moment as the `start` and `test` npm tasks don't work (because they have hard-coded reliances on `react` packages). I think we might need to `eject` from create-react-app

### CODE REVIEW

**Review by halfninja — APPROVED** [review]
Might be a time to bin off and gitignore the dist directory? Assume people are just getting it through NPM and we don't need to support serving directly from GitHub.

**Comment by matmannion** [comment]
> Might be a time to bin off and gitignore the dist directory? Assume people are just getting it through NPM and we don't need to support serving directly from GitHub.

I think we need to distribute dist versions through npm or consuming apps would need to duplicate all the react stuff for building it?

**Comment by halfninja** [comment]
> > Might be a time to bin off and gitignore the dist directory? Assume people are just getting it through NPM and we don't need to support serving directly from GitHub.
> 
> I think we need to distribute dist versions through npm or consuming apps would need to duplicate all the react stuff for building it?

I mean keep it in the prepublish task so it's included in the published package, just doesn't need to be in git

File: package.json [hunk 1]
```diff
@@ -2,24 +2,18 @@
...
   "version": "0.0.8",
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "1.2.34",
-    "@fortawesome/pro-light-svg-icons": "5.15.2",
-    "@fortawesome/react-fontawesome": "0.1.14",
     "@testing-library/jest-dom": "5.11.9",
     "@testing-library/react": "11.2.5",
     "@testing-library/user-event": "12.7.0",
-    "abortcontroller-polyfill": "1.7.1",
     "node-sass": "5.0.0",
```
Comment (halfninja) [inline package.json:11]: Can all the build and test deps go into devDependencies, particularly node-sass which a few of our app builds end up compiling from source which takes forever then never uses it.

Possibly everything but preact

**Review by lol768 — APPROVED** [review]

**Pull request merged**